sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -206,6 +213,15 @@ func runCtr(containerdSocket string, namespace string, containerID string, sourc
 
 	// If the container doesn't already exist, create it
 	if container == nil {
+		// Set the destination name for the container persistent storage location
+		var persistentPath string
+
+		if bootstrap {
+			persistentPath = "bootstrap-containers/" + containerID
+		} else {
+			persistentPath = "host-containers/" + containerID
+		}
```

Comment:
Since you're treating non-boostrap as the default, it'd be slightly clearer to write the code like this:

```go
persistentDir := "host-containers/" + containerID
if bootstrap {
	persistentDir = "bootstrap-containers/" + containerID
}
```

Reply:
I refactored the code, let me know if you have any concerns with the new approach I used